File: codeforces/practice/266A/stones.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
  string s;
  int n, k=0, ans = 0;
  cin >> n;
  cin >> s;
  
  char prev = 0;
  while (s[k]!='\0') {
    if (s[k] == prev) ans++;
    prev = s[k];
    k++;
  }
  printf("%d\n", ans);
  return 0;
}

File: codeforces/practice/266A/stones.scala
import scala.io.StdIn._

object Main {
  def main(args: Array[String]): Unit = {
    val n = readInt
    val s = readLine
    var prev = 0
    var ans = 0
    s.foreach { c =>
      if (c == prev) ans += 1;
      prev = c
    }
    println(ans)
  }
}
